custom_components/isy994/binary_sensor.py

```diff
@@ -35,18 +36,14 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def async_setup_platform(
-    hass,
-    config: ConfigType,
-    async_add_entities: Callable[[list], None],
-    discovery_info=None,
-):
+async def async_setup_entry(hass, entry, async_add_entities):
```

Comment:
Need to add types -- have to make MyPy happy for future merge with Core code.

Reply:
done